package.json

```diff
@@ -93,6 +98,7 @@
     "urijs": "1.19.11",
     "url-join": "4.0.1",
     "vite": "5.1.6",
+    "wait-port": "1.1.0",
```

Comment:
- wait for port to be ready to receive requests when launching webdriver on this port